specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -1001,6 +1070,12 @@
     }
   },
   "definitions": {
+    "TriggerEvaluationResults": {
```

Comment:
>TriggerEvaluationResults [](start = 5, length = 24)

this definition needs to define its schema

Reply:
It has no schema.  I tis just an empty response with the location header defined.

Reply:
Just removed this definition and updated the response to not reference this.